CocktailRecipes/CocktailRecipes/Features/Home/HomeViewController.swift

```diff
@@ -177,6 +189,10 @@ extension HomeViewController: SectionHeaderDelegate {
         let sectionType = presenter.sectionType(for: sectionIndex)
         let dataSource = presenter.dataSource(for: sectionType)
 
-        #warning("push to specific section here")
+        let drinkChartPresenter = DrinkChartPresenter(drinks: dataSource)
+        let drinkChartCollectionViewController = DrinkChartCollectionViewController(presenter: drinkChartPresenter)
+        drinkChartCollectionViewController.title = sectionType.title
```

Comment:
Wouldn't it be better to inject this like presenter

Reply:
Done.